providers/https/src/lib.rs

```diff
@@ -171,6 +176,53 @@ async fn check_status(config: Config) -> Result<Blob> {
 }
 
 async fn handle_query(config: Config, request: ProviderRequest) -> Result<Blob> {
+    // HACK: This KeyValueRow structure is a stop-gap measure until we
```

Comment:
Good news, we have `ArrayField` support now :)

Reply:
Not in Studio yet, we need to have a design pass to see how we want the array to appear in Studio, but we're closer than ever for sure